api/utils/mwasHypothesisGenerator.mjs

```diff
@@ -0,0 +1,117 @@
+import { getMWASResults } from "./mwas.mjs"
+import { runCypherQuery } from '../clients/neo4j.mjs';
+import { runLLMCompletion } from '../clients/oai.mjs';
+
+export const getMwasHypothesis = async (bioprojects, targetVirusFamilies, pageStart, pageEnd, identifiers) => {
+
+    let model, role;
+    model = 'gpt4o';
+    if (model === 'gpt4o') {
+        role = 'system';
+    } else {
+        role = 'assistant'; // o1
+    }
+
+
+    const results = await getMWASResults(bioprojects, targetVirusFamilies, pageStart, pageEnd);
```

Comment:
When i was testing for medium sized results, the response time was > 30 seconds for this call + LLM generation. This works fine when running locally, but our deployed version has a timeout of 30 seconds so it'll throw an error.

We'll need to pass the single MWAS row we want to generate a hypothesis for in args before merging, which is the better design.

Reply:
Now passes in selected MWAS result as parameter to base hypothesis generation on, see latest commit: d56d4bbb08682c5672d2ba3d502e6597fc476ab6